Company.App.Tests/Company.App.Application.Tests/Company.App.Application.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <RuntimeFrameworkVersion>2.1.6</RuntimeFrameworkVersion>
```

Comment:
I believe this hack is not needed here, it was needed for the OperCover + PowerShell script approach.